test/unit/workers/edition_create_link_monitor_worker_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class EditionCreateLinkMonitorWorkerTest < ActiveSupport::TestCase
+  setup do
+    @edition = create(:submitted_edition)
+    @link_monitor = CreateEditionLinkMonitor.new(@edition)
+    CreateEditionLinkMonitor.any_instance.stubs(:perform!).returns id: 1
+  end
+
+  test "#perform calls CreateEditionLinkMonitor when valid" do
+    CreateEditionLinkMonitor.expects(:new).with(@edition).returns(@link_monitor)
+
+    worker_invocation = EditionCreateLinkMonitorWorker.new.perform(@edition.id)
+
+    assert worker_invocation
+    assert_equal worker_invocation, id: 1
```

Comment:
I don't understand why this returns `{ id: 1 }`?

`EditionCreateLinkMonitorWorker#perform` returns `CreateEditionLinkMonitor#perform!` and that returns `can_perform?` which is presumably a boolean?

Reply:
Oh, nevermind. It's stubbed.

We should probably swap the order of `expected`, `actual` around.